docs/proposals/ReID-example-for-Sedna.md

```diff
@@ -0,0 +1,88 @@
+
+### Use Cases
+
+- Users can query human flow statistics from the cloud side.
+- Users can use the history human flow data recorded on the cloud to predict human flow in the future.
+- Users can query information about suspicious character through the records in the database.
+- The system can help administrators identify the name/ID of people whose information has already been stored in the database.
+
+## Design Details
+
+### Architecture and Modules
+
+<img src=".\images\ReID-example-for-Sedna-AppArch.png" style="zoom:70%;" />
+
+- `Re-ID Manager`
+
+  It exposes API so that users can invoke corresponding functions. It can interact between users and internal services, which belongs to the `Global Manager` in Sedna. The Pod requires only CPU support and is deployed in the cloud. In addition, it can output result video stream after being processed by other pods.
+
+- `FE-Re-ID Pod`
+
+  Feature extraction and Re-ID are conducted in this pod, which is based on the preprocessing results provided by edge node -- `Dectection/Tracking Pods`. At the same time, relevant functions are invoked by API in `Re-ID Manager Pod`. The Pod requires a GPU or CPU and is deployed in the cloud.
+
+- `Database`
+
+  It contains three kinds of data.
+
+  - User Feature: It is used for identity comparison in Re-ID, which may be collected in advance.
+  - Re-ID Records: It contains `time`, `cameraID`, `personID`, `direction`, etc. They are updated when people enter/exit the building, which can be used for security authentication.
+  - Human Flow Statistics: It contains `time`, `gateID`, `inNum`, `outNum`, etc. They are updated after a specific interval, such as one hour, which can be used to display the total number of persons in the building or carry out data analysis for other purpose.
+
+- `Traffic Prediction Pod`(optional)
+
+  It is an optional pod, which can be added when human flow prediction is needed. It can use the human flow statistics to train a time-series prediction model and used to predict human flow statistics in the future.
+
+- `Website / App`
+
+  It receives results provided by `Re-ID Manager` API and display them visually on the website or APP. The content includes the video processed by the algorithm, the data of human flow and so on. This module is the demonstration module of the case, which can be customized by the user.
+
+- `Detection/Tracking Pods`
+
+  It take care of object detection and tracking. specifically, it receives video stream from edge devices and provides preprocessing for `FE-Re-ID Pod` in the cloud. Video stream can come from surveillance cameras and other sources. The Pod requires GPU or CPU support and is deployed at the edge.
+
+## Road Map
+
+### 2022.07
+
```

Comment:
Raod map should not be put here, this road map is just for OSPP.

Reply:
Do you mean that I should delete this part?